recipes/libgsasl/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.52.0"
+
+
+class LibgsaslConan(ConanFile):
+    name = "libgsasl"
+    description = "Portable gsasl C library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/gsasl/"
+    license = "BSD-3-Clause"
+    topics = ("libgsasl")
+
+    description = "GNU Simple Authentication and Security Layer"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "ntlm": [False, True]
+    }
+    default_options = {"shared": False, "ntlm" : False}
+    requires = ("libiconv/1.17",
+                "libidn/1.36"
+                )
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+        try:
+            del self.settings.compiler.cppstd
+        except Exception:
+            pass
+        try:
+            del self.settings.compiler.libcxx
+        except Exception:
+            pass
+
+    def validate(self):
+        # Currently tested only on Linux platform.
+        if self.settings.os == "Windows" or self.settings.os == "Macos":
+            raise ConanInvalidConfiguration("gsasl is only supported on Linux right now.")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],
+            destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        yes_no = lambda v: "yes" if v else "no"
+        tc = AutotoolsToolchain(self)
+        tc.configure_args.append("--enable-ntlm=%s"  % yes_no(self.options.ntlm))
+        tc.generate()
+
+    def build(self):
+        autotools = Autotools(self)
+        autotools.configure()
+        autotools.make()
+
+    def package(self):
+
+        copy(self, "ABOUT-NLS", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "NEWS", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "AUTHORS", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "COPYING", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "COPYING.LIB", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "README", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "THANKS", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
```

Comment:
This should just be the licenses, I believe.

```suggestion
        copy(self, "COPYING", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
        copy(self, "COPYING.LIB", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
```